packages/utilities/src/array.ts

```diff
@@ -26,11 +26,11 @@ export function findIndex(array: any[], cb: (item: any, index?: number) => boole
  * @param size - Size of array.
  * @param getItem - Callback to populate given cell index.
  */
-export function createArray(size: number, getItem?: (index?: number) => any): any[] {
```

Comment:
Intentional breaking change to make type safety work by making getItem required. Thoughts @dzearing ?